src/re_frame/router.cljc

```diff
@@ -224,11 +224,12 @@
         ;; anything goes wrong, we know where it came from.
         ;; To get a source mapped stack, we must get rid of the react frames
         ;; See https://github.com/Day8/re-frame/issues/164#issuecomment-233528154
-        stack  (->> (js/Error. (str "Event " (first event-v) " dispatched from here:"))
-                    .-stack
-                    clojure.string/split-lines
-                    (remove #(re-find #"react.inc.js|\(native\)" %))
-                    (clojure.string/join "\n"))]
+        stack  #?(:cljs (->> (js/Error. (str "Event " (first event-v) " dispatched from here:"))
+                             .-stack
+                             clojure.string/split-lines
+                             (remove #(re-find #"react.inc.js|\(native\)" %))
```

Comment:
This wasn't you, but probably good to fix the regex to `#"react\.inc\.js|\(native\)"`